src/test/java/org/fundacionjala/coding/roger/CounterCharacterTest.java

```diff
@@ -0,0 +1,51 @@
+package org.fundacionjala.coding.roger;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test VowelVounter.
+ */
+
+public class CounterCharacterTest {
+
+    /**
+     * Final var.
+     */
+
+    public static final int ESP1 = 4;
+    public static final int ESP2 = 6;
+    public static final int ESP3 = 5;
+    public static final int ESP4 = 2;
```

Comment:
@AT-08/at-08 
These constants should be declared inside method as local final variables.

Reply:
@AT-08/at-08 
keep in mind that the variables inside test method are called 
expectedResult and actualResult